firedantic/_async/model.py

```diff
@@ -97,7 +97,12 @@ def get_document_id(self):
 
     @classmethod
     async def find(
-        cls: Type[TAsyncBareModel], filter_: Optional[dict] = None
+        cls: Type[TAsyncBareModel],
+        filter_: Optional[dict] = None,
+        order_by: Optional[
+            tuple[str, Union[Literal["ASCENDING"], Literal["DESCENDING"]]]
+        ] = None,
+        limit: Optional[int] = None,
```

Comment:
I guess you'd also want an offset if you implement limit?

Reply:
good point, i should probably add that.

our system is still in proto / early testing phase so we don't have enough data to need offsets really :D

thanks for the review and note!